src/components/box/box.stories.mdx

```diff
@@ -63,11 +63,37 @@ Basic component that gives access to a number of styling options. Should be used
   </Story>
 </Preview>
 
-## Props
+## Scroll
+Use the `scrollVariant` prop to indicate which theme scrollbar you would like to display. Use `light` or `dark` values to display as below. This will still require the use of the `overflow` prop and only displays in Chrome or Safari
+<Preview>
+  <Story name="scroll" parameters={{ info: { disable: true }}} >
+    <div>
+      <Box display='inline-block' size="150px" overflow="auto" scrollVariant="light" mr="20px">
+        <Box width="100px" height="100px" bg="primary" display="inline-block" m="5px"></Box>
+        <Box width="100px" height="100px" bg="primary" display="inline-block" m="5px"></Box>
+        <Box width="100px" height="100px" bg="primary" display="inline-block" m="5px"></Box>
+      </Box>
+      <Box display='inline-block' size="150px" overflow="auto" scrollVariant="dark">
+        <Box width="100px" height="100px" bg="primary" display="inline-block" m="5px"></Box>
+        <Box width="100px" height="100px" bg="primary" display="inline-block" m="5px"></Box>
+        <Box width="100px" height="100px" bg="primary" display="inline-block" m="5px"></Box>
+      </Box>
+    </div>
+  </Story>
+</Preview>
 
 <StyledSystemProps
   spacing
   color
   flexBox
   layout
-/>
\ No newline at end of file
+/>
+<PropsTable rows={[
+  {
+    name: "scrollVariant",
+    type: { summary: 'dark | light' },
+    description: 'String to denote which theme style to use for scrollbars, can be light theme or dark theme',
```

Comment:
```suggestion
    description: 'String to denote which theme style to use for scrollbars,
```

I think this is confusing, the prop table will say `light|dark` but the comment says it can be `light theme` or `dark theme`. I think this is better omitted.